setup_rules

```diff
@@ -0,0 +1,24 @@
+#!/bin/env sh
+# Wrapper script for flatpak
+
+runscript() {
+    echo "$@" | flatpak-spawn --env=TERM=xterm-256color --host script --quiet /dev/null 
+}
+
+runcommand() {
+    flatpak-spawn --env=TERM=xterm-256color --host $@
+}
+
+hostcopy() {
+    CAT='cat > '
+    COMMAND=$CAT$2
+    echo $COMMAND
+    cat "$1" | flatpak-spawn --env=TERM=xterm-256color --host bash -c "$COMMAND"
+}
+
+TMP=`runcommand mktemp -d`
+hostcopy /app/bin/setup_rules.sh $TMP/setup_rules.sh
+runcommand chmod +x $TMP/setup_rules.sh
+runcommand pkexec $TMP/setup_rules.sh
```

Comment:
This is not going to pass. Instructing users in appdata/elsewhere how to do it is fine, but calling `pkexec` is too much.

Reply:
I'm gonna remove that asap. I was reading in the meantime other issues/pulls/threads on the same topic how are handled in other apps.

Reply:
Removed